variables.tf

```diff
@@ -0,0 +1,185 @@
+################################################################################
```

Comment:
Could those different variables be split into different files?

Reply:
It looks that the linter don't like this ``` variable "public_gateway_bastion_enabled" should be moved from public_gateway_variables.tf to variables.tf ```